Directory.Build.props

```diff
@@ -61,6 +61,8 @@
 
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+
+    <BuildWorkloads Condition="'$(BuildWorkloads)' == ''">false</BuildWorkloads>
```

Comment:
nit: I'm not sure this is necessary

Reply:
Yup thanks, this was from an earlier iteration where `false` had specific meaning.